dev-guide.md

```diff
@@ -121,18 +128,18 @@ Once this is set up, you'll need to run a one-time database migration to set up
 
 ```
 # activate virtual environment
-source venv_smallboard/bin/activate
+pipenv shell
 
 # from smallboard/ root directory
-(venv_smallboard)$ python manage.py migrate
+(smallboard-XYZ)$ python manage.py migrate
 ```
 
 #### Local deployment
 
 Once the Python environment and database are set up and running, you can run Small Board locally using
 
 ```
-(venv_smallboard)$ python manage.py runserver
+(smallboard-XYZ)$ python manage.py runserver
```

Comment:
Hmm, actually, maybe we should just remove this virtual env parens stuff and just say that all following commands assume that you're inside a `pipenv shell` environment? It will save us from having to add extra text to everything.

I also realized there's a bunch of commands below like `heroku` (should we add this to dev deps?) and `pre-commit` that rely on the environment.

Reply:
yeah im mixed on this, it wasn't consistent to begin with. i will remove